### PR TITLE
Add app insights to FSharp template

### PR DESCRIPTION
**Pull request by satvu**
Official docs: https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide#application-insights

### CODE REVIEW

File: Functions.Templates/ProjectTemplate_v4.x/FSharp-Isolated/Company.FunctionApp.fsproj [hunk 1]
```diff
@@ -10,6 +10,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.19.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.15.1" />
+    <PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.21.0" />
```
Comment (soninaren) [inline Functions.Templates/ProjectTemplate_v4.x/FSharp-Isolated/Company.FunctionApp.fsproj:13]: Should we also be include the package to C# template?

**Review by soninaren — COMMENTED** [review]
Curious on what is driving this change.

File: Functions.Templates/ProjectTemplate_v4.x/FSharp-Isolated/Company.FunctionApp.fsproj (hunk 1 above)
Comment (satvu) [reply Functions.Templates/ProjectTemplate_v4.x/FSharp-Isolated/Company.FunctionApp.fsproj:13]: Not sure what you mean by this (haven't worked on templates before) but same change is going into the C# template in #1474.

Comment (soninaren) [reply Functions.Templates/ProjectTemplate_v4.x/FSharp-Isolated/Company.FunctionApp.fsproj:13]: Ahh i did not see that PR

**Review by soninaren — APPROVED** [review]

**Review by kshyju — APPROVED** [review]

**Comment by kshyju** [comment]
Can a netfx app be created using F#? In the other PR the change was to handle that as well.

**Comment by satvu** [comment]
> Can a netfx app be created using F#? In the other PR the change was to handle that as well.

@brettsam @mattchenderson

**Pull request merged**